src/layout/globalNav/nav/items/staging.ts

```diff
@@ -0,0 +1,111 @@
+import { INavItems } from "./INavItems";
+import { AlethioIcon } from "../../../../icon/AlethioIcon";
+import { AlethioApiIcon } from "../../../../icon/AlethioApiIcon";
+import { AlethioMonitoringIcon } from "../../../../icon/AlethioMonitoringIcon";
+import { EthStatsIcon } from "../../../../icon/EthStatsIcon";
+import { AlethioReportsIcon } from "../../../../icon/AlethioReportsIcon";
+import { DexWatchIcon } from "../../../../icon/DexWatchIcon";
+import { BriefcaseIcon } from "../../../../icon/BriefcaseIcon";
+import { InfoIcon } from "../../../../icon/InfoIcon";
+import { TwitterIcon } from "../../../../icon/TwitterIcon";
+import { GithubIcon } from "../../../../icon/GithubIcon";
+import { MediumIcon } from "../../../../icon/MediumIcon";
+import { DiscordIcon } from "../../../../icon/DiscordIcon";
+import { LinkedInIcon } from "../../../../icon/LinkedInIcon";
+
+export const navItems: INavItems = [
+    {
+        label: "navMenu.group.developerAPI",
```

Comment:
I'm thinking this should be a JSON config loaded at runtime and that it should be localized. That would also allow to localize the URLs if need be.